dwml-data-parsers/parameter.js

```diff
@@ -72,9 +72,23 @@ var parameterParser = {
 
     while (i < values.length) {
       var currentValue = values[i];
+
+      currentTimeFrame = timeFrames[timeFrameCounter];
```

Comment:
I like that you put this here, do you think you could remove it from the 2 `if`-statements below?